### PR TITLE
Add documentation for application management endpoints

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
## Proposed changes

Describes the functionality of "mobile:" endpoints designed for advanced applications management

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by imurchie — CHANGES_REQUESTED** [review]
A couple of grammatical issues.

File: docs/en/writing-running-appium/ios/ios-xctest-mobile-apps-management.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+## Advanced Applications Management Commands For iOS With WebDriverAgent/XCTest Backend
+
+Since Xcode9 there is a possibility to manage multiple applications in scope of a single session. It makes possible to open iOS preferences and change values there while the application under test is in background and then restore it back or check scenarious, where the application under test is terminated and then started again. Appium for iOS has special set of `mobile:` subcommands, which provides user interface to such features.
```
[the remark this answers is not in the document]
Comment (mykola-mokhnach) [reply docs/en/writing-running-appium/ios/ios-xctest-mobile-apps-management.md]: typo

[the remark this answers is not in the document]
Comment (mykola-mokhnach) [reply docs/en/writing-running-appium/ios/ios-xctest-mobile-apps-management.md]: fixed

File: docs/en/writing-running-appium/ios/ios-xctest-mobile-apps-management.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+`Important note`: Make sure you don't cache WebElement instances between application restarts, since they are going to be invalidated after each restart.
+
+
+### mobile: installApp
+
+Installs given application to the device under test. If the same application is already installed then it's going to be installed over it, which allows to test upgrades. Be careful while reinstalling the main application under test - make sure you called `terminateApp` for it first, otherwise WebDriverAgent will detect it as a potential application crash.
```
[the remark this answers is not in the document]
Comment (mykola-mokhnach) [reply docs/en/writing-running-appium/ios/ios-xctest-mobile-apps-management.md]: fixed

**Review by imurchie — APPROVED** [review]

**Pull request merged**